src/index.html

```diff
@@ -9,6 +9,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <a href="#home" class="nav__logo">
+          <img src="images/logo.png" alt="Logo Home">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple"
+              class="nav__link is-active">Apple</a>
```

Comment:
```suggestion
              <a 
                href="#apple"
                class="nav__link is-active"
              >
                Apple
              </a>
```
correct formatting (fix everywhere)
[Checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) (5)